test/ImportCommand.js

```diff
@@ -1,30 +1,35 @@
+import fs from "graceful-fs";
 import pathExists from "path-exists";
 import assert from "assert";
 import path from "path";
-import fs from "fs";
 import escapeArgs from "command-join";
 import mkdirp from "mkdirp";
 
 import PromptUtilities from "../src/PromptUtilities";
 import ChildProcessUtilities from "../src/ChildProcessUtilities";
 import ImportCommand from "../src/commands/ImportCommand";
-import exitWithCode from "./_exitWithCode";
-import initFixture from "./_initFixture";
-import initExternalFixture from "./_initExternalFixture";
-import assertStubbedCalls from "./_assertStubbedCalls";
+import exitWithCode from "./helpers/exitWithCode";
+import initFixture from "./helpers/initFixture";
+import initExternalFixture from "./helpers/initExternalFixture";
+import assertStubbedCalls from "./helpers/assertStubbedCalls";
 
 describe("ImportCommand", () => {
 
   describe("import", () => {
-    let testDir, externalDir;
-
-    beforeEach((done) => {
-      testDir = initFixture("ImportCommand/basic", done);
-    });
-
-    beforeEach((done) => {
-      externalDir = initExternalFixture("ImportCommand/external", done);
-    });
+    let testDir;
+    let externalDir;
+
+    beforeEach(() =>
+      Promise.resolve()
```

Comment:
This started out as a parallel `Promise.all()`, but the synchronous `process.chdir()` inside `initFixture()`'s promise chain made that non-deterministic, hence the ordering.